### ISSUE TITLE
Add a dedicated key for Taligent / IBM proprietary license

### ISSUE THREAD

**Issue by sschuberth** [nexB/scancode-toolkit#1365]
[proprietary_70.RULE](https://github.com/nexB/scancode-toolkit/blob/3f970b1e156442b869663b22e27aa8f9637c2c9e/src/licensedcode/data/rules/proprietary_70.RULE) detects findings like https://github.com/typetools/checker-framework/blob/3c427f380f61f234688893467d9bc433cf1a584c/checker/jdk/index/src/java/sun/util/resources/OpenListResourceBundle.java#L26-L39 as a generic "proprietary" license. Since we come across this often, it would be nice to give this one a separate key with a more telling name (probably including "Taligent" and / or "IBM").

**Comment by pombredanne** [comment]
Sure thing... I suggest something like this:
```
key: taligent-jdk
short_name: Taligent JDK Proprietary Notice
name: Taligent JDK Proprietary Notice
category: Proprietary Free
owner: Taligent
homepage_url: https://github.com/typetools/checker-framework/blob/3c427f380f61f234688893467d9bc433cf1a584c/checker/jdk/index/src/java/sun/util/resources/OpenListResourceBundle.java#L26-L39
notes: This notice is common in OpenJDK and related Java code.
```

and this text:
```
The original version of this source code and documentation
is copyrighted and owned by Taligent, Inc., a wholly-owned
subsidiary of IBM. These materials are provided under terms
of a License Agreement between Taligent and Sun. This technology
is protected by multiple US and International patents.

This notice and attribution to Taligent may not be removed.
Taligent is a registered trademark of Taligent, Inc.
```

**Comment by pombredanne** [comment]
@sschuberth thanks. Available in develop and 3.0.2!
